device/globals/global_vm.gd

```diff
@@ -695,7 +659,7 @@ func save():
 
 func set_camera(p_cam):
 	camera = p_cam
-	camera.clear_current()
```

Comment:
This caused the game scene camera to not be used, which I believe was a bug. Instead, the default viewport camera must have been used, which was probably the reason why extra clamping code was needed. The camera behaves much better with this camera actually being set.

Reply:
make_current() seems clearer anyway as the name is more self-explanatory.

Reply:
Looking into this, it seems to be a bug introduced by ee1d6ac106f3b9eb9bca1a56ba0ff28ac55670bb. With a custom camera it makes sense that the Escoria camera would need to be cleared, but with that gone, this line should probably have been removed. I'll update the PR.